docs/Documentation.md

```diff
@@ -1,16 +1,20 @@
----
-layout: page
-title: Documentation guide
----
+<frontmatter>
+  layout: default.md
+  title: "Documentation guide"
+  pageNav: 3
+  pageNavTitle: "Table Of Content"
+</frontmatter>
+
+# Documentation guide
 
 **Setting up and maintaining the project website:**
 
-* We use [**Jekyll**](https://jekyllrb.com/) to manage documentation.
+* We use [**MarkBind**](https://markbind.org/) to manage documentation.
 * The `docs/` folder is used for documentation.
-* To learn how set it up and maintain the project website, follow the guide [_[se-edu/guides] **Using Jekyll for project documentation**_](https://se-education.org/guides/tutorials/jekyll.html).
+* To learn how set it up and maintain the project website, follow the [_[markbind/user-guide] **MarkBind User Guide**_](https://markbind.org/userGuide/gettingStarted.html).
 * Note these points when adapting the documentation to a different project/product:
-  * The 'Site-wide settings' section of the page linked above has information on how to update site-wide elements such as the top navigation bar.
-  * :bulb: In addition to updating content files, you might have to update the config files `docs\_config.yml` and `docs\_sass\minima\_base.scss` (which contains a reference to `AB-3` that comes into play when converting documentation pages to PDF format).
+  * The [_[markbind/tweaking the page structure]_](https://markbind.org/userGuide/tweakingThePageStructure.html) and [_[markbind/site.json]_](https://markbind.org/userGuide/siteJsonFile.html) pages have information on how to update site-wide elements such as the top navigation bar.
```

Comment:
I think best not to use square brackets when giving links, as it looks like we made a mistake while using the `[text](link)` syntax.
![image](https://user-images.githubusercontent.com/1673303/218267660-e023d260-6239-4134-b5c4-d63de7dd081f.png)

Reply:
This is to follow the format used in the original docs when referencing se-edu, e.g. `[_se-edu/guides **Markdown coding standard**_](https://se-education.org/guides/conventions/markdown.html)`. I have updated the markbind references to not use the `[]`, for the se-edu ones on other pages, do you think we should remove the square brackets as well?

Reply:
OK to keep for consistency, or remove if that looks better. I'll leave the decision to you.